acquisitions/src/main/resources/global/finances.feature

```diff
@@ -172,6 +172,39 @@ Feature: global finances
     When method POST
     Then status 201
 
+  Scenario: create funds 4
```

Comment:
Maybe we can be replace these 2 with reusable features found in `thunderjet/mod-finance/reusable`?

Reply:
changed